elba.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "gallais/idris-tparsec"
+version = "0.1.0"
+authors = ["G. Allais"]
```

Comment:
I think you should be listed as a co-author.

Reply:
Ok! :)